attendees.txt

```diff
@@ -1 +1,3 @@
 Mat Harden
+
```

Comment:
I don't know, didn't think of it - maybe you told us to not add any space but I didn't hear that. Apologies for that!